app/import_functions_test.go

```diff
@@ -2885,4 +2886,31 @@ func TestImportDirectPostWithAttachments(t *testing.T) {
 	assert.Equal(t, len(attachments), 1)
 	assert.Contains(t, attachments[0].Path, "noteam")
 	AssertFileIdsInPost(attachments, th, t)
+
+	// attempt to import again with same data, should NOT add an attachment
```

Comment:
Probably would be nice to separate each test cases with `t.Run` calls, to have a clearer division of what each test case means.